### PR TITLE
Development

### PR DESCRIPTION
**Pull request by sergeysova**

### CODE REVIEW

**Comment by almindor** [comment]
Hey, sorry I missed this. Is this still relevant or do you want to keep it off?

I had it buried in my emails and forgot about it.

File: Cargo.toml [hunk 1]
```diff
@@ -6,4 +6,4 @@ authors = ["Ales Katona <almindor@gmail.com>"]
 [dependencies]
 dotenv = "0.12.0"
 postgres = "0.15.2"
-web3 = "0.3.0"
+web3 = { git = "https://github.com/akropolisio/rust-web3" }
```
Comment (almindor) [inline Cargo.toml:9]: Is this some community fork? I need to know why the switch.

**Review by almindor — COMMENTED** [review]
Looks good otherwise, can you please explain why the custom web3 implementation? I just want to be sure there's no trickery involved here :)

File: Cargo.toml (hunk 1 above)
Comment (sergeysova) [reply Cargo.toml:9]: When using the published crate, It builds fails.
I fixed it https://github.com/akropolisio/rust-web3/commit/b99a4190b246ea50735be3f633d03a12b18a6c62#diff-8e3870b5d7879691f4a03fe1f17ad3b5R35

**Comment by almindor** [comment]
Thank you!

**Pull request merged**